site/content/en/docs/user-manual/command-reference/_index.md

```diff
@@ -2,43 +2,77 @@
 title: 'Command reference'
 linkTitle: 'Command reference'
 description: ''
-weight: 7
+weight: 6
 ---
 
-> **Note**: command invocation syntax is subject to change,
-> **always refer to command --help output**
-
-Available CLI commands:
-
-<div class="text-center large-scheme">
+<div class="text-center large-scheme-two">
 
 ```mermaid
+%%{init { 'theme':'netrual' }}%%
 flowchart LR
-    d{datum}
-    p((project))
-    s((source))
-    m((model))
-    d==>p
-    p==create===>str1([Creates a Datumaro project])
-    p==import===>str2([Generates a project from other project or dataset in specific format])
-    p==export===>str3([Saves dataset in a specific format])
-    p==extract===>str4([Extracts subproject by filter])
-    p==merge===>str5([Adds new items to project])
-    p==diff===>str6([Compares two projects])
-    p==transform===>str7([Applies specific transformation to the dataset])
-    p==info===>str8([Outputs valuable info])
-    d==>s
-    s==add===>str9([Adds data source by its URL])
-    s==remove===>str10([Remove source dataset])
-    d==>m
-    m==add===>str11([Registers model for inference])
-    m==remove===>str12([Removes model from project])
-    m==run===>str13([Executes network for inference])
-    d==>c(create)===>str14([Calls project create])
-    d==>a(add)===>str15([Calls source add])
-    d==>r(remove)===>str16([Calls source remove])
-    d==>e(export)===>str17([Calls project export])
-    d==>exp(explain)===>str18([Runs inference explanation])
+  d(("#0009; datum #0009;")):::mainclass
+  s(source):::nofillclass
+  m(model):::nofillclass
+
+  d===s
+    s===id1[add]:::hideclass
+    s===id2[remove]:::hideclass
+    s===id3[info]:::hideclass
+  d===m
+    m===id4[add]:::hideclass
+    m===id5[remove]:::hideclass
+    m===id6[run]:::hideclass
+    m===id7[info]:::hideclass
+  d====str1[create]:::filloneclass
+  d====str2[add]:::filloneclass
+  d====str3[remove]:::filloneclass
+  d====str4[export]:::filloneclass
+  d====str5[info]:::filloneclass
+  d====str6[transform]:::filltwoclass
+  d====str7[filter]:::filltwoclass
+  d====str8[diff]:::fillthreeclass
+  d====str9[merge]:::fillthreeclass
+  d====str10[validate]:::fillthreeclass
+  d====str11[explain]:::fillthreeclass
+  d====str12[stats]:::fillthreeclass
+  d====str13[commit]:::fillfourclass
+  d====str14[checkout]:::fillfourclass
+  d====str15[status]:::fillfourclass
+  d====str16[log]:::fillfourclass
+
+  classDef nofillclass fill-opacity:0;
+  classDef hideclass fill-opacity:0,stroke-opacity:0;
+  classDef filloneclass fill:#CCCCFF,stroke-opacity:0;
+  classDef filltwoclass fill:#FFFF99,stroke-opacity:0;
+  classDef fillthreeclass fill:#CCFFFF,stroke-opacity:0;
+  classDef fillfourclass fill:#CCFFCC,stroke-opacity:0;
 ```
 
 </div>
+
```

Comment:
I can't see the image here:
![image](https://user-images.githubusercontent.com/13832349/132347364-92f4d2c3-0dda-46aa-a7c5-992483a49bd7.png)

No errors in the log.